tools/cordova/builder.js

```diff
@@ -115,7 +115,7 @@ export class CordovaBuilder {
       author: 'A Meteor Developer',
       email: 'n/a',
       website: 'n/a',
-      contentUrl: `http://localhost:${cordovaServerPort}/`
```

Comment:
I'll need a bit of context on this removal.

Reply:
I've rolled back, it's actually not necessary and would introduce issues on multiple apps on iOS